src/models/person.ts

```diff
@@ -28,9 +28,13 @@ export interface IAdresse {
 }
 
 export interface IBarn {
+  id?: string;
   alder: number;
   fnr: string;
   fødselsdato: string;
+  personnummer?: string;
```

Comment:
Hva er forskjellen på dette og fnr?

Reply:
Dette er de fem siste tegnene i fødselsnummeret (den delen som ikke er fødselsdato). Når man legger til et ekstra barn, har vi separerte felter for å skrive inn fødselsdato og personnummer, da det ikke er sikkert at barnet har fått personnummer ennå.